src/js/views/patients/schedule/schedule.scss

```diff
@@ -1,17 +0,0 @@
-.schedule__filters {
-  align-items: center;
-  display: flex;
-  width: 100%;
-}
-
-.schedule__all-filters:not(:empty) {
-  margin-right: 24px;
-}
-
-.schedule__filter:not(:empty) {
-  margin-right: 8px;
-}
-
-.schedule__list-search {
-  margin-left: auto;
-}
```

Comment:
I deleted this file and moved these styles over to the `schedule-list.scss` file instead. As it didn't make sense why these were separated.

But I only ended up moving `.schedule__list-search` over. 

The `.schedule__filters` class was removed from the view since it has a wrapper `div` that applies these same exact styles, so it wasn't doing anything.

And `.schedule__all-filters:not(:empty)` and `schedule__filter:not(:empty)` were removed since those aren't doing anything anymore with the changes we've made to the views.